src/RuleSetInterface.php

```diff
@@ -30,7 +30,7 @@ public static function create(array $set = array());
      *
      * @param string $rule name of rule
      *
-     * @return array|null
+     * @return array|bool
```

Comment:
Do you have a suggestion how to consistently reject undefined rules even when they are disabled without a BC break?